qutebrowser/mainwindow/prompt.py

```diff
@@ -492,6 +492,13 @@ def prompt_fileselect_external(self):
         assert len(folders) == 1
         self.prompt_accept(folders[0])
 
+    @cmdutils.register(
+        instance='prompt-container', scope='window',
+        modes=[usertypes.KeyMode.prompt, usertypes.KeyMode.yesno])
+    def prompt_toggle(self):
+        """Toggle visibility of prompts."""
```

Comment:
Maybe

```suggestion
        """Show/hide the currently shown prompt."""
```

or so, to make it a bit clearer to the user what this does.